Added some command to docs (#26)
discord-tickets/docs: docs/features/commands.md

	/add member: @eartharoid ticket: #ticket-91
	```

### `/priority`

> Sets an emoji in front of the channel name to declare the tickets priority

Reviewer: ```suggestion
Add an emoji before the channel name to declare the ticket's priority.
```

#### Options

|                          Name                            |                            Description                            |    Example    |
| :------------------------------------------------------: | :---------------------------------------------------------------: | :-----------: |

> | :material-asterisk:{ .pink title="Required" } `priority` |                  The priority to set		               |     `HIGH`    |

Reviewer: ```suggestion
| :material-asterisk:{ .pink title="Required" } `priority` |                  The new ticket priority		               |               |
```

> ??? example

Reviewer: I don't think examples are necessary for this command because there is a single required option, and it has choices to choose from rather than being free input.
